pom.xml

```diff
@@ -27,10 +27,14 @@
     </modules>
 
     <properties>
-        <assertj.version>3.9.0</assertj.version>
+        <java.version>1.8</java.version>
+        <junit.version>5.3.1</junit.version>
```

Comment:
Do we still need the junit dependency? I'm note sure.

Reply:
I forgot versions are already provided from parent POM